src/analytics/amplitude.py

```diff
@@ -99,6 +101,36 @@ def build_hit(self, res, view, request, *args, **kwargs):
         }
         hit = json.dumps(hit, default=json_serial)
         return self.forward_event(hit)
+    
+    def track_revenue_event(
```

Comment:
I would add a try/except block in the definition here because this is used in critical flows of the app and if this throws an exception so will the calling function.